### DISCUSSION TITLE
Frustrated with the "Always open" setting

### DISCUSSION THREAD

**Discussion by flappybatpal** [mozilla/multi-account-containers#2675] [General]
If I manually set "Always open" I should not be getting asked if I want to continue in my current container.
I've manually set multiple containers for a reason, not to get repeatedly asked because I dared to not open a new container tab first.

**Comment by kelimuttu** [comment]
Hey @flappybatpal, would you please open [a bug report](https://github.com/mozilla/multi-account-containers/issues/new?assignees=&labels=bug&projects=&template=bug.yml) instead?
